Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:18.10.0-alpine as builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm install && \
```

Comment:
Optimise this layer please :3

Reply:
> Optimise this layer please :3

wdym by this? it's also a build layer.

Reply:
oh do you mean splitting up package.json for faster builds?

Reply:
I meant for caching - copy the lockfiles and package.json over in a separate layer and run npm install. The actual nginx layer is what is exported so it doesn't matter if the dep install layer is separate instead.